src/components/DataIncrementsButtonForTheCalendar/DataIncrementsButtonForTheCalendar.css

```diff
@@ -0,0 +1,74 @@
+.date-selection-container {
+  display: flex;
+  column-gap: 10px; 
+  row-gap: 10px; 
+}
+
+.option {
```

Comment:
I'd recommend to remove the repetitions from this class and check which classes you really need. For example this code: 
```
margin-bottom: 0px;
margin-left: 0px;
margin-right: 0px;
margin-top: 0px;
```
can be written as:` margin: 0`

This is just one example.

Reply:
The .css is revised and optimized based on needed classes.
thanks